README.md

```diff
@@ -46,35 +50,37 @@ $('form').formPrefill({
 
 ## API
 
-The form’s API is accesible via `$('form').data('formPrefill')`.
+The form’s API is accesible via the return value of `formPrefill(form)`.
 ```javascript
 // Prefill all fields that have values saved in the stores (this is done automatically when you call the plugin on a form):
-$('form').data('formPrefill').readAll();
+let form = document.querySelector('form')
+let api = formPrefill([form])
```

Comment:
Doesn’t `formPrefill()` expect a single wrapperElement now (like in the example above)?